sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1192,3 +1192,57 @@ def test_dtype_match():
             lr_64.fit(X_64, y_64)
             assert_equal(lr_64.coef_.dtype, X_64.dtype)
             assert_almost_equal(lr_32.coef_, lr_64.coef_.astype(np.float32))
+
+
+def test_warm_start_converge_LR():
+    # Test to see that the logistic regression converges on warm start,
```

Comment:
This explanation would be improved if you referenced the issue it fixes

Reply:
Please address this comment